.github/workflows/release-package.yaml

```diff
@@ -0,0 +1,90 @@
+name: Release Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        type: choice
+        description: package
+        options:
+          - barcode
+          - channable
+          - clockify-enhanced
+          - cronhooks
+          - fulfillmenttools
+          - google-enhanced
+          - kaufland-marketplace
+          - moco
+          - otto-market
+          - sentry-io-enhanced
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  NODE_VERSION: 18.20.2
+
+jobs:
+  release-package:
+    name: Release Package
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          filter: 'blob:none'
+          fetch-depth: 0
+          fetch-tags: true
+
+      - name: Configure git User
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
```

Comment:
```suggestion
        with:
          cache: npm
```

Reply:
Added, but I have removed the "Restore Node.js modules cache" step based on [this](https://github.com/actions/cache/blob/main/examples.md#node---npm) recommendation from the `GitHub Cache Action`

> Note It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci